sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseDataView.java

```diff
@@ -211,11 +211,11 @@ protected void initView(String params) {
 		final boolean deleted = FacadeProvider.getCaseFacade().isDeleted(uuid);
 
 		if (deleted) {
-			layout.disable(CommitDiscardWrapperComponent.DELETE_UNDELETE);
+			layout.disableVithViewAllow(CommitDiscardWrapperComponent.DELETE_UNDELETE);
```

Comment:
shouldn't this be changed in all views having side pannel?

Reply:
since this ticket did not require the change of the side components behavior on all core views I've made this change in order to accommodate both behaviors. The initial behavior was that when a ticket is archived the entire side component was readOnly. The current ticket changes this behavior in order to still be able to readOnly access these components. Since some components are used in many views some aren't, in order to keep the behavior consistent I made it possible to have both behaviors at the same time until the rest of the views will be changed.